README.md

```diff
@@ -43,7 +63,7 @@ First make sure the following are installed on your system:
 Start with the following in your bash config.
 
 ```
-export PATH=$PATH:~/govuk/govuk-docker/bin
+export PATH=$PATH:/usr/local/sbin:~/govuk/govuk-docker/bin
```

Comment:
I've opened a PR #143 so users shouldn't have to do this.

Reply:
Cool, thanks 👍